DigitalLearningSolutions.Web/Helpers/DisplayStringHelper.cs

```diff
@@ -30,7 +30,9 @@ private static string GenerateBytesDisplayString(long byteCount)
             }
 
             var place = Convert.ToInt32(Math.Floor(Math.Log(byteCount, 1024)));
-            var number = Math.Round(byteCount / Math.Pow(1024, place), 1);
+            // Do not include decimal place below GB
+            var decimalPlaces = place <= 2 ? 0 : 1;
+            var number = Math.Round(byteCount / Math.Pow(1024, place), decimalPlaces);

```

Comment:
Could we write unit tests for the rounding? 
Also, I'm going to check with HEE whether we should only round only below GB or always (I don't think it's clear from the comment on the ticket)